packages/waku/src/router/create-pages.ts

```diff
@@ -175,7 +175,11 @@ export function createPages(
         page as {
           staticPaths: string[] | string[][];
         }
-      ).staticPaths.map((item) => (Array.isArray(item) ? item : [item]));
+      ).staticPaths
+        .map((item) => (Array.isArray(item) ? item : [item]))
+        .map((slug) =>
+          slug.map((i) => i.replace(/\./g, '').replace(/ /g, '-')),
+        );
```

Comment:
Is this less readable?
```suggestion
        .map((item) => (Array.isArray(item) ? item : [item]).map((i) => i.replace(/\./g, '').replace(/ /g, '-'));
```